charts/parca/templates/server-service.yaml

```diff
@@ -11,6 +11,12 @@ metadata:
   {{- end }}
 spec:
   type: {{ .Values.server.service.type }}
+  {{- with .Values.server.service.loadBalancerIP }}
```

Comment:
uh, i actually missed that there is two declarations of same block? I will fix that in master today, i think.

Reply:
sorry for that 🤦